CMakeLists.txt

```diff
@@ -154,6 +154,7 @@ find_package(LibWebSockets 4.0.1 REQUIRED)
 ##
 
 find_library(dw_lib dw DOC "libdw used to symbolize QD_MEMORY_DEBUG backtraces")
+find_package(libunwind)
```

Comment:
why not make it `REQUIRED`?

Reply:
No all platforms support libunwind I believe.  Need to discuss further with ssorj.

Reply:
In that case maybe make it optional now, and as a next step, we can add container image builds for all the platforms we care about, and we'll then see if it works.

Reply:
here's example of the multiplatform build, https://github.com/rh-messaging/cli-cpp/blob/main/.github/workflows/image_build.yaml#L61-L90

use `podman run --privileged --rm docker.io/tonistiigi/binfmt --install all` to wire up qemu

then `uses: redhat-actions/buildah-build@v2` github action takes care of the rest

(or it can be done the way @fgiorgetti did it in the skupper PR, with docker instead of podman. also there is what the amq broker does, https://github.com/artemiscloud/activemq-artemis-broker-image/tree/main using https://docs.cekit.io/en/latest/.)

Reply:
https://github.com/skupperproject/skupper-router/issues/1087